spin_coater.c

```diff
@@ -2,7 +2,7 @@
 #include "stdint.h"
 #include "hardware/timer.h"
 
-#define NUMBER_OF_WHOLES 2
+#define NUMBER_OF_HOLES 2
```

Comment:
Please amend the review changes to the original commits and not ship broken commits + fix.